### ISSUE TITLE
exit status 100 and setuidgid: usage: setuidgid account child when running Dokku Managed Cron

### ISSUE THREAD

**Issue by zuchmanski** [dokku/dokku#4416]
## Description of problem

I cannot get the new _Dokku Managed Cron_ feature work in my setup. The cron entry is successfully added to the dokku's crontab but the command itself fails to run, even for a basic `echo 1`.

```
> cat app.json

{
    "cron": [
      {
        "command": "echo 1",
        "schedule": "* * * * *"
      }
    ]
  }
```

```
> dokku cron:list panel
ID                                                                            Schedule    Command
bG9nYm94LXBhbmVsPT09ZWNobyAxPT09KiAqICogKiAq                                  * * * * *   echo 1
```

```
> sudo su dokku
> crontab -e

VPATH=/usr/local/bin:/usr/bin:/bin
SHELL=/bin/bash

* * * * * dokku --rm run --cron-id bG9nYm94LXBhbmVsPT09ZWNobyAxPT09KiAqICogKiAq panel%!(EXTRA string=echo 1)
```

When I look at the logs the only thing I can find are lines:

```
panel.run.9949 | 2021/02/19 06:38:07 exit status 100
panel.run.9949 | setuidgid: usage: setuidgid account child
panel.run.1219 | 2021/02/19 06:37:08 exit status 100
panel.run.1219 | setuidgid: usage: setuidgid account child
panel.run.6872 | 2021/02/19 06:36:07 exit status 100
panel.run.6872 | setuidgid: usage: setuidgid account child
```

Could you point me in the right direction to understand where does the second part of the command (`%!(EXTRA string=echo 1)`) comes from? I do not think it is a standard bash/dokku/cron syntax.

```
> dokku --rm run --cron-id bG9nYm94LXBhbmVsPT09ZWNobyAxPT09KiAqICogKiAq panel%!(EXTRA string=echo 1)
 !     App name must begin with lowercase alphanumeric character, and cannot include uppercase characters, colons, or underscores

> echo $?
1
```

When I replace `%!(EXTRA string=echo 1)` with `echo 1` everything works as expected.

```
> dokku --rm run --cron-id bG9nYm94LXBhbmVsPT09ZWNobyAxPT09KiAqICogKiAq panel echo 1
1

> echo $?
0
```

### `dokku report APP_NAME` output

```
ubuntu@ip-172-31-26-57:~$ dokku report panel
-----> uname: Linux ip-172-31-26-57 5.4.0-1037-aws #39~18.04.1-Ubuntu SMP Fri Jan 15 02:48:42 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
-----> memory:
                     total        used        free      shared  buff/cache   available
       Mem:            475         181          17           0         276         281
       Swap:          1999         481        1518
-----> docker version:
       Client: Docker Engine - Community
        Version:           20.10.3
        API version:       1.41
        Go version:        go1.13.15
        Git commit:        48d30b5
        Built:             Fri Jan 29 14:33:13 2021
        OS/Arch:           linux/amd64
        Context:           default
        Experimental:      true

       Server: Docker Engine - Community
        Engine:
         Version:          20.10.3
         API version:      1.41 (minimum version 1.12)
         Go version:       go1.13.15
         Git commit:       46229ca
         Built:            Fri Jan 29 14:31:25 2021
         OS/Arch:          linux/amd64
         Experimental:     false
        containerd:
         Version:          1.4.3
         GitCommit:        269548fa27e0089a8b8278fc4fc781d7f65a939b
        runc:
         Version:          1.0.0-rc92
         GitCommit:        ff819c7e9184c13b7c2607fe6c30ae19403a7aff
        docker-init:
         Version:          0.19.0
         GitCommit:        de40ad0
-----> docker daemon info:
       Client:
        Context:    default
        Debug Mode: true
        Plugins:
         app: Docker App (Docker Inc., v0.9.1-beta3)
         buildx: Build with BuildKit (Docker Inc., v0.5.1-docker)

       Server:
        Containers: 4
         Running: 3
         Paused: 0
         Stopped: 1
        Images: 6
        Server Version: 20.10.3
        Storage Driver: overlay2
         Backing Filesystem: extfs
         Supports d_type: true
         Native Overlay Diff: true
        Logging Driver: json-file
        Cgroup Driver: cgroupfs
        Cgroup Version: 1
        Plugins:
         Volume: local
         Network: bridge host ipvlan macvlan null overlay
         Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
        Swarm: inactive
        Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
        Default Runtime: runc
        Init Binary: docker-init
        containerd version: 269548fa27e0089a8b8278fc4fc781d7f65a939b
        runc version: ff819c7e9184c13b7c2607fe6c30ae19403a7aff
        init version: de40ad0
        Security Options:
         apparmor
         seccomp
          Profile: default
        Kernel Version: 5.4.0-1037-aws
        Operating System: Ubuntu 18.04.5 LTS
        OSType: linux
        Architecture: x86_64
        CPUs: 1
WARNING: No swap limit support
WARNING: No blkio weight support
WARNING: No blkio weight_device support
        Total Memory: 475.4MiB
        Name: ip-172-31-26-57
        ID: 6SRV:R7QR:JJPT:SANX:4ASX:WGQS:Y6UK:VEJR:HNM7:WLYW:YLDO:QHJP
        Docker Root Dir: /var/lib/docker
        Debug Mode: false
        Registry: https://index.docker.io/v1/
        Labels:
        Experimental: false
        Insecure Registries:
         127.0.0.0/8
        Live Restore Enabled: false

-----> git version: git version 2.17.1
-----> sigil version: 0.6.0
-----> herokuish version:
       herokuish: 0.5.25
       buildpacks:
         heroku-buildpack-multi     v1.0.0
         heroku-buildpack-ruby      v223
         heroku-buildpack-nodejs    v183
         heroku-buildpack-clojure   v86
         heroku-buildpack-python    v188
         heroku-buildpack-java      v69
         heroku-buildpack-gradle    v34
         heroku-buildpack-scala     v88
         heroku-buildpack-play      v26
         heroku-buildpack-php       v189
         heroku-buildpack-go        v151
         buildpack-nginx            v14
         buildpack-null             v3
-----> dokku version: dokku version 0.23.5
-----> plugn version: plugn: 0.6.1
-----> dokku plugins:
         00_dokku-standard    0.23.5 enabled    dokku core standard plugin
         20_events            0.23.5 enabled    dokku core events logging plugin
         app-json             0.23.5 enabled    dokku core app-json plugin
         apps                 0.23.5 enabled    dokku core apps plugin
         builder-cnb          0.23.5 enabled    dokku core builder-cnb plugin
         builder-dockerfile   0.23.5 enabled    dokku core builder-dockerfile plugin
         builder-herokuish    0.23.5 enabled    dokku core builder-herokuish plugin
         buildpacks           0.23.5 enabled    dokku core buildpacks plugin
         certs                0.23.5 enabled    dokku core certificate management plugin
         checks               0.23.5 enabled    dokku core checks plugin
         common               0.23.5 enabled    dokku core common plugin
         config               0.23.5 enabled    dokku core config plugin
         cron                 0.23.5 enabled    dokku core cron plugin
         docker-options       0.23.5 enabled    dokku core docker-options plugin
         domains              0.23.5 enabled    dokku core domains plugin
         enter                0.23.5 enabled    dokku core enter plugin
         git                  0.23.5 enabled    dokku core git plugin
         letsencrypt          0.9.2 enabled    Automated installation of let's encrypt TLS certificates
         logs                 0.23.5 enabled    dokku core logs plugin
         network              0.23.5 enabled    dokku core network plugin
         nginx-vhosts         0.23.5 enabled    dokku core nginx-vhosts plugin
         plugin               0.23.5 enabled    dokku core plugin plugin
         proxy                0.23.5 enabled    dokku core proxy plugin
         ps                   0.23.5 enabled    dokku core ps plugin
         repo                 0.23.5 enabled    dokku core repo plugin
         resource             0.23.5 enabled    dokku core resource plugin
         scheduler-docker-local 0.23.5 enabled    dokku core scheduler-docker-local plugin
         shell                0.23.5 enabled    dokku core shell plugin
         ssh-keys             0.23.5 enabled    dokku core ssh-keys plugin
         storage              0.23.5 enabled    dokku core storage plugin
         tags                 0.23.5 enabled    dokku core tags plugin
         tar                  0.23.5 enabled    dokku core tar plugin
         trace                0.23.5 enabled    dokku core trace plugin
=====> panel app information
       App deploy source:
       App dir:                       /home/dokku/panel
       App locked:                    false
=====> panel buildpacks information
       Buildpacks computed stack:     gliderlabs/herokuish:latest
       Buildpacks global stack:
       Buildpacks list:               https://github.com/heroku/heroku-buildpack-ruby.git
       Buildpacks stack:
CN = example.com
error 20 at 0 depth lookup: unable to get local issuer certificate
=====> panel ssl information
       Ssl dir:                       /home/dokku/panel/tls
       Ssl enabled:                   true
       Ssl hostnames:                 example.com www.example.com
       Ssl expires at:                Jul 31 09:18:15 2020 GMT
       Ssl issuer:                    C = US, O = Lets Encrypt, CN = Lets Encrypt Authority X3
       Ssl starts at:                 May  2 09:18:15 2020 GMT
       Ssl subject:                   subject=CN = example.com
       Ssl verified:                  self signed
=====> panel checks information
       Checks disabled list:          none
       Checks skipped list:           none
=====> panel cron information
       Cron task count:               2
=====> panel docker options information
       Docker options build:
       Docker options deploy:         --restart=on-failure:10
       Docker options run:
=====> panel domains information
       Domains app enabled:           true
       Domains app vhosts:            example.com www.example.com
       Domains global enabled:        true
       Domains global vhosts:         ip-172-31-26-57
=====> panel git information
       Git deploy branch:             master
       Git global deploy branch:      master
       Git keep git dir:              false
       Git rev env var:               GIT_REV
       Git sha:                       e84bf55
       Git last updated at:           1613678683
=====> panel logs information
       Logs computed max size:        10m
       Logs global max size:          10m
       Logs global vector sink:
       Logs max size:
       Logs vector sink:              honeycomb://?api_key=XXX&dataset=panel
=====> panel network information
       Network attach post create:
       Network attach post deploy:
       Network bind all interfaces:   false
       Network web listeners:         172.17.0.5:5000
=====> panel nginx information
       Nginx access log format:
       Nginx access log path:         /var/log/nginx/panel-access.log
       Nginx bind address ipv4:
       Nginx bind address ipv6:       ::
       Nginx client max body size:
       Nginx disable custom config:   false
       Nginx error log path:          /var/log/nginx/panel-error.log
       Nginx global hsts:             true
       Nginx computed hsts:           true
       Nginx hsts:
       Nginx hsts include subdomains: true
       Nginx hsts max age:            15724800
       Nginx hsts preload:            false
       Nginx proxy buffer size:       4096
       Nginx proxy buffering:         on
       Nginx proxy buffers:           8 4096
       Nginx proxy busy buffers size: 8192
       Nginx proxy read timeout:      60s
       Nginx last visited at:         1613717136
       Nginx x forwarded for value:   $remote_addr
       Nginx x forwarded port value:  $server_port
       Nginx x forwarded proto value: $scheme
=====> panel proxy information
       Proxy enabled:                 true
       Proxy port map:                http:80:5000 https:443:5000
       Proxy type:                    nginx
=====> panel ps information
       Deployed:                      true
       Processes:                     1
       Ps can scale:                  true
       Ps restart policy:             on-failure:10
       Restore:                       true
       Running:                       true
       Status web 1:                  running (CID: 4fe858f35f2)
=====> panel resource information
=====> panel scheduler-docker-local information
       Scheduler docker local disable chown:
=====> panel storage information
       Storage build mounts:
       Storage deploy mounts:
       Storage run mounts:
```

```
> cat Procfile
web: bundle exec puma -C config/puma.rb
release: bundle exec rails db:migrate
cron: sleep infinity
```

> This is required! Issues missing this information may be closed.
>
> For problems affecting all applications, the report output for a broken application is useful for our debugging.
> In these cases, you may run `dokku report` without any arguments to display the top-level reporting information.

### Additional information

- Container Inspect Output (if applicable) via `dokku ps:inspect APP_NAME`
- The nginx configuration (if applicable) via `dokku nginx:show-config APP_NAME`
- Link to the exact repository being deployed (if possible/applicable):
- If a deploy is failing or behaving unexpectedly:
  - Application name
  - The type of application being deployed (node, php, python, ruby, etc.)
  - If using buildpacks, which custom buildpacks are in use
  - If using a `Dockerfile`, the contents of that file
  - If it exists, the contents of your `Procfile`.
- Output of failing Dokku commands after running `dokku trace:on`
  (BEWARE: `trace:on` will print environment variables for some commands, be sure you're not exposing any sensitive information when posting issues. You may replace these values with XXXXXX):

**Comment by josegonzalez** [comment]
This was fixed in 0.23.6, please upgrade.

**Comment by zuchmanski** [comment]
@josegonzalez Perfect, thank you!
